### ISSUE TITLE
[BUG] Bot does not handle .torrent files

### ISSUE THREAD

**Issue by carcade** [noam09/deluge-telegramer#23]
**Describe the bug**
Bot fails adding download, when sending .torrent file.

**To Reproduce**
1. Send .torrent file to bot

**Expected behavior**
Bot adds new download.

**Screenshots**
<img width="532" alt="Снимок экрана 2020-04-12 в 12 19 55" src="https://user-images.githubusercontent.com/16689838/79062957-f933c380-7cb7-11ea-91af-fd7e5aa709ba.png">

**OS and Version:**
 - OS: Linux Ubuntu server 18.04
 - Deluge version: 1.3.15
 - Bot version: v1.2.1-alpha

**Additional context**
You can use ubuntu image torrent file as example - http://releases.ubuntu.com/18.04/ubuntu-18.04.4-desktop-amd64.iso.torrent

**Comment by noam09** [comment]
Hi @carcade, does this also happen when using the latest alpha release? ([Telegramer-1.2.1.7-alpha](https://github.com/noam09/deluge-telegramer/releases/download/v1.2.1-alpha/Telegramer-1.2.1.7-alpha-py2.7.egg))

**Comment by carcade** [comment]
> Hi @carcade, does this also happen when using the latest alpha release? ([Telegramer-1.2.1.7-alpha](https://github.com/noam09/deluge-telegramer/releases/download/v1.2.1-alpha/Telegramer-1.2.1.7-alpha-py2.7.egg))

Yes, I've used exactly this file from this release post - https://github.com/noam09/deluge-telegramer/releases/tag/v1.2.1-alpha

**Comment by noam09** [comment]
I tried downloading the .torrent file you linked using Telegramer, and it worked successfully. Nevertheless there is definitely a bug there if it's not working for you, we just need to understand what exactly.
Can you post a Deluge debug log from the point you `/add` and up until the download fails?

**Comment by carcade** [comment]
Hmm.. today it just have stopped replying anything when I send .torrent files. But it correctly replies to commands. Deluge log shows nothing about this.
Maybe I have to enable some verbose log level?
P.S. I'm running deluge in docker container

**Comment by carcade** [comment]
So, in addition to previous comment: 
when I send .torrent file from desktop telegram client (macos) - bot replies "Aw man... That's not a torrent file",
when I send .torrent file from mobile telegram client (ios) - bot replies nothing.

In both cases there are no deluge logs appeared.

**Comment by noam09** [comment]
Hey, I somehow missed your replies. Please try posting a debug log as such:

> Please attach a debug log, make sure to censor out any sensitive info.
> `deluge -L debug -l <logfile>`

So if you're running a Docker container, you may need to map a volume for the log file (`-v`), and pass the `-L debug` parameter when running. 
In a `docker-compose.yml` file this would be something like (this may contain errors, but similar to this):
```
---
version: "2.1"
services:
  deluge:
    image: linuxserver/deluge
    container_name: deluge
    # ...
    environment:
      # If you're using a LinuxServer image
      - DELUGE_LOGLEVEL=error #optional
    volumes:
      - /path/to/deluge/config:/config
      - /path/to/your/logs:/logs
    command: -l /logs/debug.log
```

**Comment by noam09** [comment]
Re-open if needed.

**Comment by Slavik763** [comment]
Hey. I've got the same problem.
<img width="695" alt="Screenshot 2023-04-05 at 21 45 08" src="https://user-images.githubusercontent.com/38501567/230192662-5ab26987-47b5-49e9-986f-ff20288fba61.png">

Here is a log https://pastebin.com/LtBtaw3t
Not sure if it's relevant to this problem but it's all I've got.

deluged 2.0.3
libtorrent: 2.0.5.0
Python: 3.10.6
OS: Linux Ubuntu 22.04 Jammy Jellyfish
Telegramer-2.1.1.0-py3.10.egg (builded with setup.py on the machine).

Bot itself seems to be works fine.
